src/main.rs

```diff
@@ -176,6 +176,12 @@ fn main() {
                 .short("i")
                 .help("Ignore non-zero exit codes."),
         )
+        .arg(
+            Arg::with_name("capture-out")
```

Comment:
I would prefer `capture-output` here. Also, I'd like to remove the `-c` short version because this is really only meant for debugging purposes.